srcpkgs/renderdoc/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'renderdoc'
+pkgname=renderdoc
+version=1.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_GL=ON -DENABLE_GLES=ON -DENABLE_PYRENDERDOC=ON
+-DENABLE_QRENDERDOC=ON -DENABLE=RENDERDOCCMD=ON -DENABLE_VULKAN=ON
+-DENABLE_XCB=ON -DENABLE_XLIB=ON"
+makedepends="libX11-devel libxcb-devel xcb-util-keysyms-devel libGL
+python3-devel qt5-devel qt5-svg-devel qt5-x11extras-devel pcre-devel"
+hostmakedepends="bison autoconf automake  pkg-config"
```

Comment:
hostmakedepends= should be directly above makedepends=

Reply:
and python3 seems to be needed in `hostmakedepends` according to https://travis-ci.org/void-linux/void-packages/jobs/453012614

Reply:
What should I do about `WARNING: renderdoc-1.1_1: librenderdoc.so not found in common/shlibs`?

Reply:
Unless you believe some other package needs to link against that library, nothing. That warning is just a reminder that you might have forgotten to add the library to `common/shlibs` but if nothing else links to it, that's not necessary anyway.